contributors/devel/sig-instrumentation/migration-to-structured-logging.md

```diff
@@ -467,16 +477,17 @@ func ChangePodStatus(newStatus, currentStatus string) {
 ## Good practice for passing values in structured logging
 
 When passing a value for a key-value pair, please use following rules:
-* Prefer using Kubernetes objects (for example `*v1.Pod`) and log them using `klog.KObj`
+* Prefer using Kubernetes objects (for example `*v1.Pod`) and log them using `klog.KObj` or `klog.KObjSlice`
```

Comment:
`KObjSlice` is better off with resources having more than one entries. Using that for `*v1.Pod` for example is probably not a good idea right ?